SPECS/prism/prism.spec

```diff
@@ -3,20 +3,19 @@
 
 Summary:        Azure Linux image customization tool
 Name:           prism
-Version:        0.12.0
+Version:        0.13.0
 Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://github.com/microsoft/azure-linux-image-tools
 Source0:        https://github.com/microsoft/azure-linux-image-tools/archive/refs/tags/v%{version}.tar.gz#/azure-linux-image-tools-%{version}.tar.gz
```

Comment:
Is there a plan to make source drops available as part of upstream releases? This certainly isn't the only spec that uses github-provided source archives as sources, but there have been [previous events](https://github.blog/open-source/git/update-on-the-future-stability-of-source-code-archives-and-hashes/) where GitHub has changed the contents of the archives over time, and they seem to strongly recommend use of release archives if there's a desire to be able to confirm that the contents of the release haven't changed.